dynawo/sources/Models/Modelica/Dynawo/Resources/Include/logEvent.h

```diff
@@ -14,10 +14,10 @@
 #include "ModelicaUtilities.h"
 
 extern void addLogEvent1(int key){};
-extern void addLogEvent2(int key, const char * arg1, const char * arg2){};
-extern void addLogEvent3(int key, const char * arg1, const char * arg2, const char * arg3){};
-extern void addLogEvent4(int key, const char * arg1, const char * arg2, const char * arg3, const char * arg4){};
-extern void addLogEvent5(int key, const char * arg1, const char * arg2, const char * arg3, const char * arg4, const char * arg5){};
+extern void addLogEvent2(int key, const char * arg1){};
```

Comment:
Why this modification? This is how the macro works

Reply:
We had several examples failing under OpenModelica as a result of an inadequate number of arguments, for instance Dynawo.Examples.Wind.IEC.Neplan.WT4ACurrentSource2015.

Reply:
By the way, addlogEvent1 needed a key but no argument, addLogEvent2 one key and two arguments, addLogEvent3 one key and three arguments.

I have changed addLogEvent2 to one key and one argument, a combination which did not exist. A better naming could be addLogEvent0 (one key, 0 argument), addLogEvent1 (one key, one argument), addLogEventN (one key, N arguments).

Reply:
You can check it on the master branch.

Reply:
ok my bad, there do is a mistake in this file and it should not jeopardize the others usage of it, we can keep this correction :)